masu/external/accounts/network/cur_accounts_network.py

```diff
@@ -0,0 +1,114 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Database source impelmentation to provide all CUR accounts for CURAccounts access."""
+
+import requests
+
+from masu.config import Config
+from masu.external.accounts.cur_accounts_interface import (CURAccountsInterface,
+                                                           CURAccountsInterfaceError)
+
+
+# pylint: disable=too-few-public-methods
+class CURAccountsNetwork(CURAccountsInterface):
+    """Provider interface defnition."""
+
+    def __init__(self):
+        """Initializer."""
+        self.koku_host = Config.KOKU_HOST
+        self.koku_port = str(Config.KOKU_PORT)
+        self.admin_user = Config.KOKU_ADMIN_USER
+        self.admin_pass = Config.KOKU_ADMIN_PASS
+
+    def _get_service_admin_token(self):
+        """
+        Retrieve service admin token from Koku.
+
+        This will call koku's token-auth to get the service admin token.
+
+        Args:
+            None
+
+        Returns:
+            (String) : Service Admin token
+
+        """
+        token_auth_url = 'http://{}:{}/api/v1/token-auth/'.format(self.koku_host, self.koku_port)
```

Comment:
We should probably support protocol as an environment variable also, so that we can encrypt the password over HTTPS in the future.

Reply:
Yes.  I can make a note in the code about this.  There are definitely more security related changes needed to be made here.  @adberglund and I were discussing using OpenShift secrets, etc.   The plan was to just keep things simple right now until we prove out the micro-service changes and then come back and do the security work once we (if we) decide to make micro service the default option.

Reply:
Maybe I'm not understanding the comment but where are koku API paths currently defined in masu?  I prefer being able to set the various components as their own environment variables, that makes it easier to change individual pieces rather than having to manipulate a longer base URL string.  Also, don't forget that this is an experimental change, not enabled by default, so I don't want to get bogged down in these types of details too much..